tests/test_config_parsing/test_ert_config_parsing.py

```diff
@@ -491,6 +491,31 @@ def test_that_define_statements_with_less_than_one_argument_raises_error():
         _ = ErtConfig.from_file(test_config_file_name)
 
 
+@pytest.mark.usefixtures("use_tmpdir")
+def test_that_define_statements_with_more_than_one_argument():
+    test_config_file_name = "test.ert"
+    test_config_contents = dedent(
+        """
+        NUM_REALIZATIONS  1
+        DEFINE <TEST1> 111 222 333
+        DEFINE <TEST2> <TEST1> 444 555
+        """
+    )
+    with open(test_config_file_name, "w", encoding="utf-8") as fh:
+        fh.write(test_config_contents)
+
+    ert_config = ErtConfig.from_file(test_config_file_name)
```

Comment:
Maybe explicitly add `use_new_parser=False` here? That default will be changed at some point, which will affect this test.